src/app/containers/RecentVideoEpisodes/index.test.jsx

```diff
@@ -66,4 +74,15 @@ describe('RecentAudioEpisodes', () => {
     expect(hiddenDuration).toBeDefined();
     expect(hiddenDuration).toContainHTML('15:00');
   });
+
+  describe('on amp', () => {
+    it('should use amp-img rather than img', () => {
+      const { container } = render(
+        <RecentVideoEpisodesWithContext episodes={recentVideoFixtures} onAmp />,
```

Comment:
Hey did this pass? Should it not be `isAmp`?

If it was passing - unless I'm missing something, then something else could be up?

Reply:
oops - good catch - it wasn't passing (https://github.com/bbc/simorgh/runs/1650979936)

Fixed now!